metaserver/src/main/scala/scala/meta/languageserver/WorkspaceCommand.scala

```diff
@@ -10,6 +10,7 @@ case object WorkspaceCommand extends Enum[WorkspaceCommand] {
 
   case object ClearIndexCache extends WorkspaceCommand
   case object ResetPresentationCompiler extends WorkspaceCommand
+  case object ScalafixUnusedImports extends WorkspaceCommand
```

Comment:
Would it be possible (in the future) to adapt Scalafix rewrites to code actions so that it won't be necessary to define a `WorkspaceCommand` for each type of rewrite?

Reply:
Scalafix absolutely, scalafix can run custom rules from configuration in .scalafix.conf, we can probably embed that config as command arguments to get away with only a single `Scalafix` WorkspaceCommand.